app/models/setting.py

```diff
@@ -209,6 +223,7 @@ def __init__(self,
         self.email_from = email_from
         self.email_from_name = email_from_name
         self.frontend_url = frontend_url
+        self.cookie_policy = cookie_policy
```

Comment:
You have added `cookie_policy` and `cookie_policy_link` in factories but __init__ function is only adding `cookie_policy` IMO this is the reason for travis dredd failure. Also if you added both attrs in schema then there is need to update blueprints too

Reply:
Oh okay I will do that :+1: